test/trimCharsStart.js

```diff
@@ -0,0 +1,49 @@
+import { assert } from 'chai';
+
+import * as RA from '../src';
+
+describe('trimCharsStart', function() {
+  it('should remove all occurences of a trim character', function() {
+    assert.strictEqual(RA.trimCharsStart('-', '---abc'), 'abc');
+  });
+
+  it('should remove all occurences of all trim characters', function() {
+    assert.strictEqual(RA.trimCharsStart('_-', '-_-abc'), 'abc');
+  });
+
+  it('should not remove the trim characters from the end of the string', function() {
+    assert.strictEqual(RA.trimCharsStart('_-', '-_-abc-_-'), 'abc-_-');
+  });
+
+  it('should curry', function() {
```

Comment:
can you pls move this test to be the last  ? We have this convention of moving currying tests to be last. Thank you!